src/lib/viewers/controls/annotations/__tests__/AnnotationsControls-test.tsx

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 import { act } from 'react-dom/test-utils';
 import { ReactWrapper, mount } from 'enzyme';
 import AnnotationsControls from '../AnnotationsControls';
+import { AnnotationColor } from '../../../../AnnotationModule';
 import { AnnotationMode } from '../types';
 
 describe('AnnotationsControls', () => {
-    const getWrapper = (props = {}): ReactWrapper => mount(<AnnotationsControls {...props} />);
+    const getWrapper = (props = {}): ReactWrapper =>
```

Comment:
Since `annotationColor` is optional and defaulted, is this change needed?

Reply:
Nope we don't need this anymore.